### ISSUE TITLE
Failed startup due to existing symlink after `docker-compose.yml` changes

### ISSUE THREAD

**Issue by fongd** [jacobalberty/unifi-docker#366]
The latest `beta` tag fails to start the controller service if I make a change to a running UniFi container. It seems like a symlink at `/usr/lib/unifi/run/unifi` tries to get created but fails if it already exists, preventing startup of the controller service.

I can reproduce this with an already running controller by changing one of the service's parameters in my `docker-compose.yml` file, running `docker-compose up -d` to apply the changes, then waiting for the controller to come up.

I think there needs to be a check to prevent the symlink from getting created if it already exists or maybe just delete the existing one before the new one gets created.

### Host operating system

Ubuntu 20.04.1 LTS

### What tag are you using

`beta`

### What complete docker command or docker-compose.yml do you use to launch the container (omitting sensitive values)?

```
unifi:
  image: jacobalberty/unifi:beta
  container_name: unifi
  volumes:
    - /opt/dockerdata/unifi:/unifi
  networks:
    net_lan:
  environment:
    - TZ=${TZ}
    - RUNAS_UID0=false
    - UNIFI_UID=1003
    - UNIFI_GID=1003
    - PKGURL=https://dl.ui.com/unifi/5.14.23-70ddb4015a/unifi_sysvinit_all.deb
  restart: unless-stopped
```

### What do you expect to happen?

UniFi should start up :)

### What actually happens?

UniFi fails to start with the following error:

```
…
ln: failed to create symbolic link '/usr/lib/unifi/run/unifi': File exists
[2020-10-17 12:06:29,620] <docker-entrypoint> Starting unifi controller service.
```

**Comment by jacobalberty** [comment]
I am trying to reproduce and so far have been unable to. I believe this may be caused by the work I'm doing for #362 . But for me I get the error then unifi starts as normal.

**Comment by fongd** [comment]
This appears to be a problem on my end. What I had forgotten in all of this is that around the same time the Docker image was updated, I set up Traefik as a reverse proxy. Strangely, if I recreate the UniFi container, Traefik works until the next time Traefik or the controller get restarted. At this point, any page I would normally be able to see in the controller after successful authentication results in a "fatal error" (400: Bad Request) page. BUT the controller is running and is communicating with ui.com so I'm closing this. Sorry for the wild goose chase!
